e2e-tests/tests/gr-tls-cert-manager/06-renew-certs.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 10
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      deploy_cmctl
+
+      current_generation_mysql=$(kubectl -n ${NAMESPACE} get sts gr-tls-cert-manager-mysql -o jsonpath='{.metadata.generation}')
+      current_generation_router=$(kubectl -n ${NAMESPACE} get deployment gr-tls-cert-manager-router -o jsonpath='{.metadata.generation}')
+
+      sleep 10
```

Comment:
why do we need sleep here?

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/548/commits/cddce2cbe750e8181b0a3cc97e8b069b99d91081